modules/ProtocolBrowser/src/client/ProtocolBrowser.coffee

```diff
@@ -442,6 +443,140 @@ class ProtocolBrowserController extends Backbone.View
 		$(".bv_protocolBaseControllerContainer").addClass("hide")
 		$(".bv_noMatchesFoundMessage").addClass("hide")
 
+	downloadSELFile: =>
+		# To generate the SEL file, we need the endpoint data from a different protocol data object we make a request for here: 
+		$.ajax
+			type: 'GET'
+			url: "/api/protocols/codename/" + @protocolController.model.escape('codeName')
```

Comment:
Just to confirm, we didn't already have the "data column order" states in @protocolController.model ?

Reply:
We do have the "data column order" states, but we don't have the protocol project information in the @protocolController.model as far as I can see which we need.

Reply:
No longer needed, logic moved to backend